clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -518,18 +518,32 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Updates a Spinnaker Server Group (Openstack Heat Stack).
+   * @param stackName
+   * @param heatTemplate
```

Comment:
The params here no longer match the signature. Also, can you comment on the differences between template and subtemplate? Including one being a string, one a map.